pkg/stdlib/rows_test.go

```diff
@@ -370,9 +370,13 @@ func TestRowsAffected_LastInsertId(t *testing.T) {
 		er: &schema.SQLExecResult{
 			Txs: []*schema.CommittedSQLTx{
 				{
+					UpdatedRows: 1,
 					LastInsertedPKs: map[string]*schema.SQLValue{
 						"table1": {Value: &schema.SQLValue_N{N: 1}},
 					},
+					FirstInsertedPKs: map[string]*schema.SQLValue{
```

Comment:
some validation on this new attribute may be missing

Reply:
what you mean? It make no sense add a new method to return FirstInsertId if in standard lib LastInsertId is used in FirstInsertId...